src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -284,9 +285,14 @@ def maybe_calibrate_or_quantize(
     }:
         return value
 
+    if value.numel() == 0:
+        # if the tensor is empty,
+        # skip quantization
+        return value
+
     if args.dynamic:
         # dynamic quantization - get scale and zero point directly from observer
-        observer = getattr(module, f"{base_name}_observer")
+        observer = getattr(module, f"{base_name}_{ObserverTypes.BASE.value}")
```

Comment:
might want to standardize `base_name` as well since these are still loose values, especially since you've moved the combined values in to the enum